reporter/build.gradle.kts

```diff
@@ -88,6 +88,7 @@ dependencies {
     api(project(":model"))
 
     implementation(project(":downloader"))
+    implementation(project(":spdx-utils"))
```

Comment:
commit-msg: I believe only mentioning `the new SpdxDocumentReporter` is a bit mis-leading, since also several other reporters (if not even all of them) do have this dependency.

Reply:
True, fixed.